src/modules/common/components/LanguageNotAvailablePage.js

```diff
@@ -23,14 +23,16 @@ const Wrapper = styled.View`
 type PropsType = {
   theme: ThemeType,
   city: string,
+  currentLanguage: string,
   languages: Array<LanguageModel>,
-  changeLanguage: (city: string, language: string) => void,
+  changeLanguage: (city: string, newLanguage: string) => void,
```

Comment:
Why do we need `city` for changing the language?

Reply:
Because if the language is not available we need to do a `navigateToXY` since there is no route mapping in the category state yet and a city is required for it.